values.yaml

```diff
@@ -146,6 +146,18 @@ core:
   additionalVolumeMounts: []
   terminationGracePeriodSeconds: 300
 
+  restore:
```

Comment:
Please comment this out in values.yaml and leave it as an example of usage, but don't hard-wire this in to the default values, can't accept it in this way.

Finally -- you can actually create a file under deployment-examples if you wish that lets you hard-wire all of the stuff that you need in a restore scenario.  In this way we can provide an example of how to use restore without having it be default behavior.

Reply:
I added an example also there is already an existing example in the restore document.